src/tests/compiler/function/parameterDeclarationTests.ts

```diff
@@ -129,4 +130,25 @@ describe(nameof(ParameterDeclaration), () => {
             doTest("function identifier(param, param2, param3) {}", "param2", "function identifier(param, param3) {}");
         });
     });
+
+    describe(nameof<ParameterDeclaration>(d => d.getStructure), () => {
+        function doTest(code: string, expectedStructure: ParameterDeclarationStructure) {
+            const {descendant} = getInfoFromTextWithDescendant<ParameterDeclaration>(code, SyntaxKind.Parameter);
+            const structure = descendant.getStructure();
+            expect(structure).to.contain(expectedStructure);
```

Comment:
We should do an exact comparison in case anything changes. `expect(structure).to.deep.equal(expectedStructure)`.

Reply:
sure, BTW right now for quickly testing I'm not doing what I've should, this is asserting on the structure object itself instead of re-render a new node asserting against the node.getText() - but at least it gives me some confidence I'm doing things OK and that it works ll the way down to text.